cmd/buildah/run.go

```diff
@@ -15,10 +15,46 @@ import (
 
 var (
 	runFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "add-host",
+			Usage: "Add a custom host-to-IP mapping (host:ip) (default [])",
```

Comment:
We try to start our Usage statements lower case.   "add a custom...".  Please adjust through out.

Reply:
fixed